tests/phpunit/CRM/Case/XMLProcessor/ProcessTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+require_once 'CiviTest/CiviCaseTestCase.php';
+
+/**
+ * Class CRM_Case_PseudoConstantTest
+ * @group headless
+ */
+class CRM_Case_XMLProcessor_ProcessTest extends CiviCaseTestCase {
+
+  public function setUp() {
+    parent::setUp();
+
+    $this->defaultAssigneeOptionsIds = array();
```

Comment:
CiviCRM recently dropped support for PHP 5.3, which means that now they can use short array syntax. In fact, I've seen they migrating to that in the most recent PRs, so you might want to do that too.